scenarios/tracking/02_evaluating_mot_challenge.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<ol><li>Download [links] to <code>MOT_SAVED_PATH</code> ("to the path" is redundant)</li><li>5GB, 2GB</li><li>"Note that it may take some time to download."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/569/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>